### PR TITLE
Remove statute feature

### LINKED ISSUE #3085: Deactivate statute feature
The feature statute amendments can be removed. It's never used any more.
Recommended way to create statute amendments: Copy full statute in a normal motion (with special category). Now, user can create (normal) amendments to this statute/motion. You get the full support for diff/amendent/pdf/projection. No special function for statute amendments. So please remove old code of statute amendments. BTW: the feature currrently not work (`NullInjectorError` after try to open list view).

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #3085 
This feature is no longer in use and is deactivated for a while. Remove it complete with sub parts.

### CODE REVIEW

**Review by Elblinator — CHANGES_REQUESTED** [review]
client-1            | ✖ Failed to compile.

File: client/src/app/gateways/repositories/motions/motion-repository.service/motion-repository.service.ts [hunk 1]
```diff
@@ -246,7 +246,6 @@ export class MotionRepositoryService extends BaseAgendaItemAndListOfSpeakersCont
             reason: partialMotion.reason,
             number: partialMotion.number,
             state_extension: partialMotion.state_extension,
-            statute_paragraph_id: partialMotion.statute_paragraph_id,
```
Comment (Elblinator) [inline client/src/app/gateways/repositories/motions/motion-repository.service/motion-repository.service.ts:249]: Please remove the whole function createStatuteAmendment(partialMotion: Partial<Motion & { workflow_id: Id }>)(see line 326)
This function is twice in the code, also amendment-controller.service.ts
and remove the state 'AmendmentAction.CREATE_STATUTEBASED_AMENDMENT'

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.ts [hunk 2]
```diff
@@ -186,32 +173,6 @@ export class MotionContentComponent extends BaseMotionDetailChildComponent {
...
-     * The paragraph of the statute to amend was changed -> change the input fields below
-     *
-     * @param {number} newValue
-     */
-    public onStatuteParagraphChange(newValue: number): void {
-        const selectedParagraph = this.statuteParagraphs.find(par => par.id === newValue);
-        this.contentForm.patchValue({
-            title: this.translate.instant(`Statute amendment for`) + ` ${selectedParagraph!.title}`,
-            text: selectedParagraph?.text,
-            workflow_id: this.getWorkflowIdForCreateFormByParagraph(newValue)
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.ts:211]: remove getWorkflowIdForCreateFormByParagraph(...) (see line 452)

**Review by Elblinator — CHANGES_REQUESTED** [review]

File: client/src/app/gateways/repositories/motions/motion-repository.service/motion-repository.service.ts (hunk 1 above)
Comment (reiterl) [reply client/src/app/gateways/repositories/motions/motion-repository.service/motion-repository.service.ts:249]: done

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.ts (hunk 2 above)
Comment (reiterl) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.ts:211]: done.

**Review by Elblinator — APPROVED** [review]
for code and function

**Comment by Elblinator** [comment]
I looked at it some more and I haven't found any wrong behaviour
I'll merge this PR now, 
While testing the new version we should click a bit more on the motion-list and motion-block list

**Pull request merged**